socialhome/streams/templatetags/slurp_json_context.py

```diff
@@ -0,0 +1,36 @@
+from django.utils.safestring import mark_safe
+from django.template import Library
+import json
+
+register = Library()
+
+
+@register.simple_tag(takes_context=True)
+def slurp_json_context(context):
+    """
+    Serializes ``context["json_context"]`` into JSON and generates a JS
+    script to attach it to ``window.context``
+    Args:
```

Comment:
Docstrings args etc should preferably be `reST` style, see from here how: https://stackoverflow.com/a/24385103/1489738

Also, Python convention is to structure docstrings like this:

```
"""This is one sentence describing the method.

This is a longer description which can span many lines and has many sentences or code.
There should be an empty line before the main body part. The closing """ is on a new line,
unless the whole docstring is just one line, then on the same line. An empty line before :param: etc.

:param foo: Nice param
"""
```

See also PEP-257: https://www.python.org/dev/peps/pep-0257/